cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/12-affording-your-loan.html

```diff
@@ -0,0 +1,147 @@
+<section class="college-costs_tool-section affording" data-tool-section="affording-your-loans">
+    <h2>Can I afford my payment?</h2>
+
+    <p> Skipping monthly loan payments can cause financial and legal problems. Default—missing too many payments—can result in low credit scores, additional fees, wage garnishment, and ineligibility for further federal student aid. This page gives you two ways to estimate the affordability of your projected monthly payment.</p>
+
+    <div id="affording-your-loans_chart"></div>
+
+    {{ number_callout.render({
+            'header': 'Estimated monthly loan payment',
+            'data_attribute': 'data-financial-item="debt_tenYearMonthly"',
+            'additional_classes': 'number-callout__large number-callout__border-top',
+            'value': '$612.94'
+        }) 
+    }}
+    <p>Your payment will depend on your total debt and your repayment plan. (This tool assumes a standard plan, where you pay the same amount every month for 10 years; other plans with lower monthly payments (and higher total costs of borrowing) are available.)</p>
+
+    {{ number_callout.render({
+            'header': 'Median monthly salary',
+            'data_attribute': 'data-financial-item="salary_monthly"',
+            'additional_classes': 'number-callout__large number-callout__border-top',
+            'value': '$4,016.67'
+        }) 
+    }}
+    <p>Your actual monthly salary will depend on many different factors, including your field of work, and the area where you will live. To get more information about the salary levels of different careers, visit the <a href="">BLS Wage Data</a> website.</p>
```

Comment:
href missing here (see other BLS comment)

Reply:
Updated it here since the href was already there...